src/Sarif.UnitTests/Core/FileDataTests.cs

```diff
@@ -72,7 +73,7 @@ public void FileData_PersistFileContentsBinary()
                 fileData.Hashes.Should().BeNull();
 
                 string encodedFileContents = Convert.ToBase64String(File.ReadAllBytes(filePath));
-                fileData.Contents.Text.Should().Be(encodedFileContents);
+                fileData.Contents.Binary.Should().Be(encodedFileContents);
```

Comment:
`fileData.Contents.Text.Should().BeNull();`

Reply:
Ah yup